lib/BackendClient.js

```diff
@@ -389,13 +411,15 @@ var BackendClient = (function(){
         client.liveStream.update(null, entryObj.entryId, KalturaMediaEntry);
         client.entryServerNode.listAction(null, EntryServerNodeFilter);
         client.liveStream.registerMediaServer(null, entryObj.entryId, mediaServerHostname, entryObj.serverType, applicationName, streamState);
+        client.liveStream.createRecordedEntry(null, entryObj.entryId, entryObj.serverType, streamState);
```

Comment:
why are you calling createRecordedEntry?  registerMediaServer should do it for you